lib/rubocop/cop/style/redundant_begin.rb

```diff
@@ -65,7 +65,8 @@ def on_def(node)
         def on_block(node)
           return if target_ruby_version < 2.5
 
-          return if node.send_node.stabby_lambda?
+          send_node = node.send_node
```

Comment:
You might also mention here why `send_node` could return `nil` for the sake of clarity.